compliance_checker/cf/util.py

```diff
@@ -573,6 +573,16 @@ def is_vertical_coordinate(var_name, var):
         satisfied |= getattr(var, "positive", "").lower() in ("up", "down")
     return satisfied
 
+def string_from_var_type(variable):
```

Comment:
This is needed for char arrays where the `_Encoding` attribute is not declared, which is virtually always, in my experience.   Still, assuming UTF-8 may have pitfalls, so if you have better ideas please let me know.